siliconcompiler/toolscripts/ubuntu22/install-openroad.sh

```
@@ -13,35 +13,30 @@ fi
 cd openroad
 git checkout $(python3 ${src_path}/_tools.py --tool openroad --field git-commit)
 git submodule update --init --recursive
 
-git config user.email "bot@siliconcompiler.com"
-git config user.name "SC-Install"
-git revert -m 1 --no-edit 3548df8222cbd2ef4a4b2900e720948205791eed
+sudo ./etc/DependencyInstaller.sh -bazel -prefix="$BAZEL_PREFIX"
+sudo chown -R $USER:$USER $BAZEL_PREFIX
 
-deps_args=""
 if [ ! -z ${PREFIX} ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)